src/config/Config.zig

```diff
@@ -7987,7 +7987,8 @@ pub const QuickTerminalSize = struct {
             tag: Tag,
             value: Value,
 
-            pub const Tag = enum(u8) { none, percentage, pixels };
```

Comment:
fyi: this introduced in https://github.com/ghostty-org/ghostty/commit/17f7f204e1478a3ae00be2644bc04865539defd6